vue-components/src/components/Lookup.vue

```diff
@@ -52,15 +52,14 @@ import { MenuItem } from '@/components/MenuItem';
  *
  * Uses the following components internally: Input, ValidationMessage and LookupMenu
  */
-export default Vue.extend( {
+export default ( Vue as VueConstructor<Vue & { $refs: { menu: InstanceType<typeof LookupMenu> } }> ).extend( {
```

Comment:
I think we might need to find a more readable solution to this in the near future.

Reply:
I think there were ideas for that in the links I read about it:

* vuejs/vue-class-component#94
* bluecanvas/vue-typed-refs

They are also in the commit message, added here for future reference